google-cloud-security_center/samples/notification.rb

```diff
@@ -42,10 +43,12 @@ def create_notification_config org_id:, config_id:, pubsub_topic:
     notification_config: notification_config
   )
   puts "Created notification config #{config_id}: #{response}."
-  # [END scc_create_notification_config]
+# [END scc_create_notification_config]
+# [END securitycenter_create_notification_config]
```

Comment:
Please retain the indentation. Also in a few other cases below.

Reply:
Done